File: src/test/java/com/twsg/lnl/kotlin/c3_extensions/g1_java/StringUtilTest.java
package com.twsg.lnl.kotlin.c3_extensions.g1_java;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

/**
 * @author tonyl
 */
public class StringUtilTest {

    @Test
    public void testCountVowels() throws Exception {
        assertEquals(3, StringUtil.countVowels("Jonathan"));
    }

}

File: src/test/java/com/twsg/lnl/kotlin/c3_extensions/g2_kotlin/StringUtilTest.kt
package com.twsg.lnl.kotlin.c3_extensions.g2_kotlin

import org.junit.Assert.assertEquals
import org.junit.Test

/**
 * @author tonyl
 */
class StringUtilTest {

    @Test
    @Throws(Exception::class)
    fun testCountVowels() {
        assertEquals(3, "Jonathan".countVowels())
    }

}